configs/config.yaml

```diff
@@ -1,3 +1,7 @@
+hydra:
+  job:
+    chdir: True
```

Comment:
This is what need to be added to suppress hydra-core warnings.

Reply:
Which warnings? If they are related to deprecated features, we should probably upgrade to the latest and fix the differences

Reply:
The warnings are related to changing the default directory of the job. The default behavior of new versions of hydra-core is that the default directory does not change (so everything will be saved to "./runs/...", not to "./outputs/runs/..."